src/algorithms/models/graph.ts

```diff
@@ -33,8 +33,26 @@ export default class Graph {
      */
     getVertex(id: number): Vertex {
         if (!this.vertices.has(id)) {
-            throw new Error('Vertex ID not found!');
+            throw new Error('Vertex ' + id + ' not found!');
         }
         return this.vertices.get(id);
     }
+
+    /**
+     * Get function which returns the number of vertices in the graph
+     */
+    getSize(): number {
+        return this.vertices.size;
+    }
```

Comment:
Why this method? I do not see it used anywhere. It also might not work well because the Graph Simplifier made by me does not remove the vertices from the Graph it just bypasses them and leaves them in the map.

Should I remove them from the map? @josef-kriz @jediflamaster

Reply:
I removed it, it's not needed. It's a relic of the past when I tried a different approach that sucked.